WalletWasabi.Gui/Controls/WalletExplorer/TransactionViewerView.xaml

```diff
@@ -12,37 +12,37 @@
-        <DockPanel LastChildFill="True">
-          <Button Margin="0 0 0 10" DockPanel.Dock="Top" Command="{Binding OpenTransactionBroadcaster}" Content="Open Transaction Broadcaster" />
-          <Button DockPanel.Dock="Bottom" Command="{Binding ExportBinaryPsbt}" Content="Export Binary PSBT" />
-        </DockPanel>
-      </Grid>
-      <Grid Classes="content" ScrollViewer.VerticalScrollBarVisibility="Auto">
-        <Grid.RowDefinitions>
-          <RowDefinition Height="26" />
-          <RowDefinition Height="1*" />
-          <RowDefinition Height="1*" />
-          <RowDefinition Height="1*" />
-        </Grid.RowDefinitions>
-        <StackPanel Grid.Row="0" Orientation="Horizontal" Margin="0 8 0 0" Spacing="5">
-          <TextBlock Text="Transaction ID:" FontWeight="Bold" />
-          <controls:ExtendedTextBox Text="{Binding TxId, ConverterParameter=50, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" Classes="selectableTextBlock" />
+    <Grid Classes="content">
+      <DockPanel LastChildFill="True">
+        <StackPanel DockPanel.Dock="Bottom" HorizontalAlignment="Right" Orientation="Horizontal" Spacing="10">
+          <Button Command="{Binding OpenTransactionBroadcaster}" Content="Open Transaction Broadcaster" />
+          <Button Command="{Binding ExportBinaryPsbt}" Content="Export Binary PSBT" />
         </StackPanel>
-        <DockPanel LastChildFill="True" Grid.Row="1" Margin="0 8 0 0">
-          <TextBlock DockPanel.Dock="Top" Text="PSBT Json" FontWeight="Bold" />
-          <controls:MultiTextBox Margin="0 8 0 0" Text="{Binding PsbtJsonText, ConverterParameter=1536, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" CopyOnClick="{Binding !IsLurkingWifeMode}" IsSelectable="False" IsReadOnly="True" TextWrapping="Wrap" AcceptsReturn="True" />
-        </DockPanel>
-        <DockPanel LastChildFill="True" Grid.Row="2" Margin="0 8 0 0">
-          <TextBlock DockPanel.Dock="Top" Text="Transaction Hex" FontWeight="Bold" />
-          <controls:MultiTextBox Margin="0 8 0 0" Text="{Binding TransactionHexText, ConverterParameter=382, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" CopyOnClick="{Binding !IsLurkingWifeMode}" IsSelectable="False" IsReadOnly="True" TextWrapping="Wrap" AcceptsReturn="True" />
-        </DockPanel>
-        <DockPanel LastChildFill="True" Grid.Row="3" Margin="0 8 0 0">
-          <TextBlock DockPanel.Dock="Top" Text="PSBT Base64 String" FontWeight="Bold" />
-          <controls:MultiTextBox Margin="0 8" VerticalAlignment="Stretch" Text="{Binding PsbtBase64Text, ConverterParameter=472, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" CopyOnClick="{Binding !IsLurkingWifeMode}" IsSelectable="False" IsReadOnly="True" TextWrapping="Wrap" AcceptsReturn="True" />
-        </DockPanel>
-      </Grid>
-    </DockPanel>
+        <Grid ScrollViewer.VerticalScrollBarVisibility="Auto">
+          <Grid.RowDefinitions>
```

Comment:
This could be simplified into a single row definition property:
`<Grid ... RowDefinitions="26,1*,1*,1*"...`

Reply:
I luv it!